### ISSUE TITLE
Histograms are not collected in parallel

### ISSUE THREAD

**Issue by mcgibbon** [ai2cm/ace#375]
The histogram aggregator currently reports results for the root rank only.

This may be a no-fix issue, since it adds a fair bit of complexity to keep the distributed bins consistent across ranks and because our histograms already appear to capture tails quite well, and because this doesn't affect standalone inference (where we run in serial). But it could definitely be fixed.

**Comment by oliverwm1** [comment]
Yeah, good to be aware of this but agreed not a high priority to fix.

**Comment by oliverwm1** [comment]
Removed wontfix label, since this is higher priority for downscaling training/evaluation.
